airbyte-ci/connectors/metadata_service/lib/metadata_service/models/src/ConnectorMetadataDefinitionV0.yaml

```
@@ -113,3 +113,5 @@ properties:
         "$ref": AirbyteInternal.yaml
       remoteRegistries:
         "$ref": RemoteRegistries.yaml
+      supportsRefreshes:
+        type: boolean
```

Should we add a sane default?
```suggestion
        type: boolean
        default: false

```

---

done